file_operations.c

```diff
@@ -0,0 +1,189 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#define FILENAME "users.txt"
+
+typedef struct
+{
+    int id;
+    char name[50];
+    int age;
+} User;
+
+void createFile()
+{
+    FILE *file = fopen(FILENAME, "a");
+    if (file == NULL)
+    {
+        perror("Unable to create file");
+        exit(EXIT_FAILURE);
+    }
+    fclose(file);
+}
+
+void addUser()
+{
+    User user;
+    FILE *file = fopen(FILENAME, "a");
+    if (file == NULL)
+    {
+        perror("Unable to open file");
+        return;
+    }
+
+    printf("Enter user ID: ");
+    scanf("%d", &user.id);
+    printf("Enter user name: ");
+    scanf("%s", user.name);
+    printf("Enter user age: ");
```

Comment:
can we handle read input and printing output in separate function? take care at all places.